### PR TITLE
bugfix/LIVE 1356

### PR DESCRIPTION
**Pull request by tlaforge-ledger**
## 🦒 Context (issues, jira)

[LIVE-1356]

## 💻  Description / Demo (image or video)

Steps to Reproduce:
Go to LLD Manager
Open app on device

Expected behavior:
An asset with the latest connected device model is shown

Actual behavior:
A Nano S asset is always shown (static asset)

When on manager, if you disconnect the nano for example, opening an app on your nano device, it uses the Disconnect component. On this component, we show a nanoX image if it's a nano X else a Nano S.
With new device LNS+ or old blue one, it's just not handled.

So I added new assets thanks to design team. We handle dark theme assets too for nano S, X, SP and blue.
You can use the env : OVERRIDE_MODEL_ID to set the model you want to test.

image
image

## 🖤  Expectations to reach

PR must pass CI, rebase develop if conflicts. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked:
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [*] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

[LIVE-1356]: https://ledgerhq.atlassian.net/browse/LIVE-1356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tlaforge-ledger** [comment]
HODL cause possible nano Blue images are coming.

**Review by thomasrogerlux — APPROVED** [review]
Code looks good to me! Maybe nano S shouldn't be default as it's getting replaced by S plus soon?

File: src/renderer/screens/manager/Disconnected.js [hunk 1]
```diff
@@ -3,39 +3,64 @@ import React, { useEffect, useState, useCallback } from "react";
...
+    width: 290,
+  },
+  nanoSP: {
+    light: nanoSP,
+    dark: nanoSPDark,
+    width: 332,
+  },
+  blue: {
+    light: blue,
+    dark: blue,
```
Comment (juan-cortes) [inline src/renderer/screens/manager/Disconnected.js:48]: Blue dark?

**Review by juan-cortes — CHANGES_REQUESTED** [review]
Just have to change the icon for blue dark, you used the light one, and we're good

**Review by juan-cortes — APPROVED** [review]

**Pull request merged**